### PR TITLE
First version of the tegolySIGN connector

### PR DESCRIPTION
**Pull request by JanGosda**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by JanGosda** [comment]
@microsoft-github-policy-service agree [company="tegoly GmbH"]

**Comment by JanGosda** [comment]
@microsoft-github-policy-service agree company="tegoly GmbH"

**Comment by Amjed-Ayoub** [comment]
Hi @JanGosda thank you so much for submitting your connector files. 
Can you please confirm if you are an Independent Publisher(You don't own the API) or Verified Publisher(You own the API)?
If you are an Independent publisher, please add the (Independent publisher) to your pull request title and move the files to the independent publishers folder. Otherwise, if you are a verified publisher, as we ask all our partners to register their connector, may I ask you please to register your connector for certification by filling out our [registration form](https://aka.ms/ConnectorRegistration).
Please let me know if you have any questions.
Thank you so much

**Comment by JanGosda** [comment]
> Hi @JanGosda thank you so much for submitting your connector files. Can you please confirm if you are an Independent Publisher(You don't own the API) or Verified Publisher(You own the API)? If you are an Independent publisher, please add the (Independent publisher) to your pull request title and move the files to the independent publishers folder. Otherwise, if you are a verified publisher, as we ask all our partners to register their connector, may I ask you please to register your connector for certification by filling out our [registration form](https://aka.ms/ConnectorRegistration). Please let me know if you have any questions. Thank you so much

Hi, we are verified publisher. I have filled out the form. Greetings

File: certified-connectors/tegolySIGN/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "tegolySIGN",
+    "description": "Automate your signatures with the tegolySIGN connector. You can transform manual processes into fully digital ones. Integrate tegolySIGN functions into your favourite applications and create fully automated signature workflows.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/tegolySIGN/apiDefinition.swagger.json:5]: @JanGosda all files should free of grammatical errors and typos. please change "favourite" to "favorite"

File: certified-connectors/tegolySIGN/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+            }
+          }
+        },
+        "description": "Returns the signed document, name and certificate"
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
```
Comment (Amjed-Ayoub) [inline certified-connectors/tegolySIGN/apiDefinition.swagger.json:48]: @JanGosda The 'default' response should not have schema definition. Schemas should be defined on expected responses only.

File: certified-connectors/tegolySIGN/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      "post": {
+        "security": [
+          {
+            "oauth2_auth": []
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
```
Comment (Amjed-Ayoub) [inline certified-connectors/tegolySIGN/apiDefinition.swagger.json:103]: @JanGosda The 'default' response should not have schema definition. Schemas should be defined on expected responses only.

File: certified-connectors/tegolySIGN/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+            },
+            "document": {
+              "type": "string",
+              "format": "byte",
+              "description": "The signed PDF document"
+            },
+            "certificate": {
+              "type": "string",
+              "format": "byte",
+              "description": "The certificate for the signed doucment"
```
Comment (Amjed-Ayoub) [inline certified-connectors/tegolySIGN/apiDefinition.swagger.json:38]: @JanGosda all files should free of grammatical errors and typos. please change "doucment" to "document"

File: certified-connectors/tegolySIGN/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "parameters": [
+          {
+            "name": "file",
+            "in": "formData",
+            "description": "The uploaded file data",
+            "required": true,
+            "type": "file"
```
Comment (Amjed-Ayoub) [inline certified-connectors/tegolySIGN/apiDefinition.swagger.json:120]: @JanGosda The 'x-ms-summary' property is required, please add to the parameters.

File: certified-connectors/tegolySIGN/apiProperties.json [hunk 6]
```diff
@@ -0,0 +1,52 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "a2b9e741-b3c9-49ab-b278-1f2c21eacf5c",
```
Comment (Amjed-Ayoub) [inline certified-connectors/tegolySIGN/apiProperties.json:8]: @JanGosda please replace the actual data with dummy data "\<<Enter your client ID\>>"

File: certified-connectors/tegolySIGN/apiProperties.json [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+          "constraints": {
+            "required": "false",
+            "hidden": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#316bac",
+    "capabilities": [],
+    "publisher": "tegoly GmbH"
```
Comment (Amjed-Ayoub) [inline certified-connectors/tegolySIGN/apiProperties.json:50]: @JanGosda please add the stackOwner property. 
"stackOwner": "Name of the owner of the service or product." should be the same of Publisher in your case.

File: certified-connectors/tegolySIGN/readme.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+## tegolySIGN Connector
```
Comment (Amjed-Ayoub) [inline certified-connectors/tegolySIGN/readme.md:1]: Please follow this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) for the readme file and for the raw view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md). please include all required sections.

File: certified-connectors/tegolySIGN/readme.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+Automate your signatures with the tegolySIGN connector. You can transform manual processes into fully digital ones. Integrate tegolySIGN functions into your favourite applications and create fully automated signature workflows.
```
Comment (Amjed-Ayoub) [inline certified-connectors/tegolySIGN/readme.md:2]: @JanGosda all files should free of grammatical errors and typos. please change "favourite" to "favorite"

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @JanGosda,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**